recipe/meta.yaml

```diff
@@ -1,21 +1,23 @@
 {% set name = "distributed" %}
-{% set version = "2023.3.2.1" %}
+{% set version = "2023.4.1" %}
+{% set build_number = "0" %}
```

Comment:
It's a code duplication because instead of 2 lines there will be 4 lines and checksum and build_number occur only 1 time in the meta.yaml. It will make sense if they will occur more than 1 time